simapp/simd/cmd/testnet.go

```diff
@@ -288,16 +287,16 @@ func initGenFiles(
 	}
 
 	authGenState.Accounts = accounts
-	appGenState[authtypes.ModuleName] = clientCtx.JSONMarshaler.MustMarshalJSON(authGenState)
+	appGenState[authtypes.ModuleName] = clientCtx.JSONMarshaler.MustMarshalJSON(&authGenState)
 
 	// set the balances in the genesis state
 	var bankGenState banktypes.GenesisState
 	clientCtx.JSONMarshaler.MustUnmarshalJSON(appGenState[banktypes.ModuleName], &bankGenState)
 
 	bankGenState.Balances = genBalances
-	appGenState[banktypes.ModuleName] = clientCtx.JSONMarshaler.MustMarshalJSON(bankGenState)
+	appGenState[banktypes.ModuleName] = clientCtx.JSONMarshaler.MustMarshalJSON(&bankGenState)
 
-	appGenStateJSON, err := codec.MarshalJSONIndent(clientCtx.JSONMarshaler, appGenState)
+	appGenStateJSON, err := json.MarshalIndent(appGenState, "", "  ")
```

Comment:
Same comments as https://github.com/cosmos/cosmos-sdk/pull/7069#discussion_r471501061

Reply:
I've been thinking of reintroducing the `liveness.sh` script.

Reply:
btw we had a quick chat about this on last Friday's call, here are some notes: https://hackmd.io/r_1h9vzrREmRrjZwWvndTg. No strong decision has been made, the bash script was one idea.

Just to get your POV, what does the bash script test that [this](https://github.com/cosmos/cosmos-sdk/blob/574c7d2d7f19bda04aa58226634870323521f17d/simapp/simd/cmd/cmd_test.go#L13) wouldn't test?